core/src/event_id.rs

```diff
@@ -146,6 +146,12 @@ impl From<&Vec<u8>> for EventId {
     }
 }
 
+impl From<&str> for EventId {
```

Comment:
This should be a TryFrom with checks that the event id as valid. Or added to EventId as `from_str_unchecked`.

Reply:
I was using this to make EventIDs for the test but now we that we
impl Key for Bytes
we just use Bytes for the test key and I can remove this.